mock/py/mockbuild/config.py

```diff
@@ -202,6 +203,14 @@ def setup_default_config_opts():
             'requires': ['preproc-rpmspec'],
             'cmd': '/usr/bin/preproc-rpmspec %(source_spec)s --output %(target_spec)s',
         },
+        'rpmautospec_enable': False,
```

Comment:
Hmm, I’m thinking if we shouldn’t set this to `True`, otherwise we need to change the code in Koji calling `mock --buildsrpm`, too.

Reply:
We may need to discuss that with Pavel; I didn't want to make any changes to the default configuration on my own.